### PR TITLE
switch from raw endpoint

### PR DESCRIPTION
**Pull request by Wattenberger**
to avoid cors issues

### CODE REVIEW

File: src/hooks/index.tsx [hunk 1]
```diff
@@ -183,22 +183,6 @@ export const useLocalStorage = (key: string, initialValue: any) => {
   return [storedValue, setValue];
 };
 
-export async function getRepoInfo(params: RepoContext): Promise<string> {
```
Comment (Wattenberger) [inline src/hooks/index.tsx:186]: lil dead code cleanup - unrelated

File: src/hooks/index.tsx [hunk 2]
```diff
@@ -74,7 +74,7 @@ export async function getFileContent(
   const { repo, owner, path, fileRef } = params;
   const branch = fileRef || "HEAD";
 
-  const apiUrl = `https://raw.githubusercontent.com/${owner}/${repo}/${branch}/${path}`;
+  const apiUrl = `https://api.github.com/repos/${owner}/${repo}/contents/${path}?ref=${branch}`;
   const res = await fetch(apiUrl, {
     headers: PAT ? { Authorization: `token ${PAT}` } : {},
   });
```
Comment (jaked) [inline src/hooks/index.tsx:80]: I think you need to unpack the `content` field from the response and base64-decode it, as we do in `getFileContent` in `blocks`. Or maybe there is a way to get raw content by giving a content-type header, see https://docs.github.com/en/rest/reference/repos#custom-media-types-for-repository-contents

Comment (Wattenberger) [reply src/hooks/index.tsx:80]: this is actually already returning raw content (I swear I tested it this time!)

Comment (Wattenberger) [reply src/hooks/index.tsx]: I'll add the header though, just to make sure!

Comment (jaked) [reply src/hooks/index.tsx]: ha ha I believe you! but I am confused, is this not the same endpoint we call in `getFileContent`? there we call it through Axios, maybe Axios sets a header that changes the response?

Comment (jaked) [reply src/hooks/index.tsx]: through curl I could only get a raw response with an `Accept` header, e.g.
```
curl -H "Accept: application/vnd.github.v3.raw" https://api.github.com/repos/githubnext/blocks-tutorial/contents/README.md
```

Comment (Wattenberger) [reply src/hooks/index.tsx]: yeah, super strange! I'm not sure how that was working for me. Lmk if the changes work for you!

**Pull request merged**